### PR TITLE
feat: Apply isort

### LINKED ISSUE #7: PEP8 Violation
According to PEP8 guidelines, I believe constants (e.g. the help lines) should follow the imports.

### PR DESCRIPTION
**Pull request by matthewfeickert**
Resolves #7 

Address the points of Issue #7 and automatically make sure this issue does not come up again in the future by using [`isort`](https://pycqa.github.io/isort/) to sort imports and force them to the top of their Python files.

**Suggested squash and merge commit message:**
```
* Add isort to pre-commit hooks
   - Add isort config options to pyproject.toml
* Apply isort to all Python files
   - Alphabetically sort imports with one import per line
* Move all imports to the top of Python files
* Add VS Code workspace setting to gitignore
```

### CODE REVIEW

**Comment by dguest** [comment]
this looks great. When can we merge it? It would fix #49 too.

**Comment by matthewfeickert** [comment]
> this looks great. When can we merge it? It would fix #49 too.

I'm going to rebase this once we finish PR #50 and then we should be good to go.

File: .pre-commit-config.yaml [hunk 1]
```diff
@@ -3,4 +3,7 @@ repos:
     rev: 20.8b1
     hooks:
     - id: black
-      language_version: python3
```
Comment (matthewfeickert) [inline .pre-commit-config.yaml:6]: This is actually redundant (I've now learned) so I've removed it here.

**Review by dguest — COMMENTED** [review]
This seems pretty ok to me. Is sorting imports alphabetically something that most of the python community recommends or is it just to reduce entropy? 

I'm fine with it either way, but of course there are other conventions for organizing imports (i.e. grouping by logical function or putting local modules before global ones etc).

**Comment by matthewfeickert** [comment]
> This seems pretty ok to me. Is sorting imports alphabetically something that most of the python community recommends or is it just to reduce entropy?
> 
> I'm fine with it either way, but of course there are other conventions for organizing imports (i.e. grouping by logical function or putting local modules before global ones etc).

I'm not sure on the history of `isort` and its choices but it is becoming more mainstream to use it given that it is part of the [Python Code Quality Authority](https://github.com/PyCQA) and people like to offload decisions about code to other people when possible (at least I like to offload it to their tools).

The sorting is done alphabetically by [sections which you can configure](https://pycqa.github.io/isort/docs/configuration/options/#sections), but I've just gone with the default for now. As an example of what the current config in `pyproject.toml` will do the is the before `isort`

```python
from my_lib import Object

import os

print("Hey")
print("yo")


from my_lib import Object3

from my_lib import Object2

import sys

from pandamonium import version

from third_party import (
    lib15,
    lib1,
    lib2,
    lib3,
    lib4,
    lib5,
    lib6,
)

import sys

from __future__ import absolute_import

from third_party import lib3
```

and here is the after

```python
from __future__ import absolute_import

import os
import sys

from my_lib import Object
from my_lib import Object2
from my_lib import Object3
from third_party import lib1
from third_party import lib2
from third_party import lib3
from third_party import lib4
from third_party import lib5
from third_party import lib6
from third_party import lib15

from pandamonium import version

print("Hey")
print("yo")
```

I'm happy to play around with import order config to get things that "seem right" if you will. @dguest do you have any preferences when it comes to the ordering of these "[sections](https://pycqa.github.io/isort/docs/configuration/options/#sections)"?

**Comment by dguest** [comment]
I don't really have a better idea about how to organize things so this is fine to me. I didn't see where this hooks into CI, how do we test to make sure this sorting is respected?

**Comment by matthewfeickert** [comment]
> I don't really have a better idea about how to organize things so this is fine to me. I didn't see where this hooks into CI, how do we test to make sure this sorting is respected?

At the moment it isn't running CI as I forgot it, but it is in the `pre-commit` hooks, so if a developer installs `pre-commit` they get picked up for free. I've made Issue #53 to fix me missing adding to CI (I forgot that we don't have [`pre-commit.ci`](https://pre-commit.ci/) enabled for this repo).

**Pull request merged**